jetty-annotations/src/main/java/module-info.java

```diff
@@ -25,6 +25,8 @@
     requires transitive org.eclipse.jetty.plus;
     requires transitive org.objectweb.asm;
     requires org.slf4j;
+    requires org.eclipse.jetty.servlet;
+    requires org.eclipse.jetty.webapp;
```

Comment:
@sbordet I'm not sure these are needed, they should be transitive through org.eclipse.jetty.plus?

Reply:
Not needed, remove them.